README.md

```diff
@@ -1,34 +1,62 @@
-[![Build Status](https://travis-ci.org/kuzzleio/sdk-go.svg?branch=master)](https://travis-ci.org/kuzzleio/sdk-go) [![codecov.io](http://codecov.io/github/kuzzleio/sdk-php/coverage.svg?branch=master)](http://codecov.io/github/kuzzleio/sdk-go?branch=master) [![GoDoc](https://godoc.org/github.com/kuzzleio/sdk-go?status.svg)](https://godoc.org/github.com/kuzzleio/sdk-go)
-[![Go Report Card](https://goreportcard.com/badge/github.com/kuzzleio/sdk-go)](https://goreportcard.com/report/github.com/kuzzleio/sdk-go)
-
-Official Kuzzle GO SDK with wrappers for C++ and JAVA SDK
-======
-
-## About Kuzzle
-
-A backend software, self-hostable and ready to use to power modern apps.
-
-You can access the Kuzzle repository on [Github](https://github.com/kuzzleio/kuzzle)
-
-* [SDK Documentation](https://godoc.org/github.com/kuzzleio/sdk-go)
-* [Installation](#installation)
-* [Basic usage](#basic-usage)
-* [Running tests](#tests)
-* [License](#license)
-
-## SDK Documentation
-
-The complete SDK documentation is available [here](https://docs.kuzzle.io/sdk-reference)
-
-## Installation
-
-````sh
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/7868838/53850936-31e57180-3fbd-11e9-8392-8f3e26bf2aa8.png"/>
+</p>
+<p align="center">
+  <img src="https://img.shields.io/badge/tested%20on-linux%20%7C%20osx%20%7C%20windows-blue.svg">
+  <a href="https://travis-ci.org/kuzzleio/sdk-go">
+    <img src="https://travis-ci.org/kuzzleio/sdk-go.svg?branch=master"/>
+  </a>
+  <a href="https://codecov.io/gh/kuzzleio/sdk-go">
+    <img src="https://codecov.io/gh/kuzzleio/sdk-go/branch/master/graph/badge.svg" />
+  </a>
+  <a href="https://goreportcard.com/report/github.com/kuzzleio/sdk-go">
+    <img src="https://goreportcard.com/badge/github.com/kuzzleio/sdk-go" />
+  </a>
+  <a href="https://godoc.org/github.com/kuzzleio/sdk-go">
+    <img src="https://godoc.org/github.com/kuzzleio/sdk-go?status.svg"/>
+  </a>
+  <a href="https://github.com/kuzzleio/sdk-go/blob/master/LICENSE">
+    <img alt="undefined" src="https://img.shields.io/github/license/kuzzleio/sdk-go.svg?style=flat">
+  </a>
+</p>
+
+## About
+
+### Kuzzle Go
+
+This is the official Go SDK for the free and open-source backend Kuzzle. It provides a way to dial with a Kuzzle server from Go applications.
```

Comment:
I prefer to let the explanatory parts about SDK Client -> Kuzzle communication. Since it's present on the Javascript SDK and Rust SDK and team up voted it.